source/exec/docker.d

```diff
@@ -79,10 +93,15 @@ class Docker: IExecProvider
 			--queueSize_;
 
 		auto encoded = Base64.encode(cast(ubyte[])source);
+		// try to find the compiler in the available images
+		auto r = DockerImages.find!(d => d.canFind(compiler));
```

Comment:
I think if `compiler == "beta"`, this will match `dmd-beta`, which I'm not sure if it's desired.
Perhaps simply:
```D
auto r = DockerImages.find(compiler);
```
?